Incrementing
showing image: Videos/2exampleImg.jpg
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.11/tkinter/__init__.py", line 1948, in __call__
    return self.func(*args)
           ^^^^^^^^^^^^^^^^
  File "/home/IntExp/Documents/vlcWork/SlideShow.py", line 43, in increment_media_index
    self.display_media()
  File "/home/IntExp/Documents/vlcWork/SlideShow.py", line 53, in display_media
    self.show_image(media_path)
  File "/home/IntExp/Documents/vlcWork/SlideShow.py", line 79, in show_image
    self.label.config(image=photo)
  File "/usr/lib/python3.11/tkinter/__init__.py", line 1702, in configure
    return self._configure('configure', cnf, kw)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/tkinter/__init__.py", line 1692, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: invalid command name ".!frame.!label"
